npm_and_yarn/lib/dependabot/npm_and_yarn.rb

```diff
@@ -56,6 +56,10 @@ module NpmAndYarn
     # Used to check if error message contains timeout fetching package
     TIMEOUT_FETCHING_PACKAGE_REGEX = %r{(?<url>.+)/(?<package>[^/]+): ETIMEDOUT}
 
+    ESOCKETTIMEDOUT = /(?<package>.*?): ESOCKETTIMEDOUT/
```

Comment:
where is this error used?

Reply:
@abdulapopoola , i have updated rest of code.